### PR TITLE
Eng 1736 final redhat release

### PR DESCRIPTION
**Pull request by ampie**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-controller-coordinator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr66**

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr66**

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | c46a27e01018129633ecca087bb4a63a3f3fad42 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 38b2331c61a651dd0a85ab1af3a0095d22ef4ec7 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | dcfa8af565acbe31d3aeb590efc45dc0ab9c6e7e | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr66**

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 39eb329803320d980dc1af36b48aa684d7b6f08e | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr66**

File: src/main/bash/copy-images.sh
Comment (w-caffiero-entando) [inline src/main/bash/copy-images.sh:11]: @ampie `${1} => "${version}" ?`

File: src/main/bash/copy-images.sh
Comment (w-caffiero-entando) [inline src/main/bash/copy-images.sh:9]: @ampie wonder if you are aware that this is not going to catch http error statuses but only network errors

you need something line this for that purpose

```
res="$(curl -f -o versions.yaml https: -o versions.yaml -w "%{http_code}" ..)"
if [ "$res" ...
```

File: src/main/bash/copy-images.sh
Comment (w-caffiero-entando) [inline src/main/bash/copy-images.sh:15]: @ampie are you missing an `echo` here and below?

File: src/main/bash/generate-csv-fragments-images.sh
Comment (w-caffiero-entando) [inline src/main/bash/generate-csv-fragments-images.sh:6]: @ampie same as above, are you really looking just for network errors?

File: src/main/bash/copy-images.sh
Comment (ampie) [reply src/main/bash/copy-images.sh:9]: The manual says: "-f, --fail          Fail silently (no output at all) on HTTP errors"
I have actually been testing that locally and it fails against 404 errors because the tag wasn't in Github yet.

File: src/main/bash/copy-images.sh
Comment (ampie) [reply src/main/bash/copy-images.sh:15]: Indeed

File: src/main/bash/generate-csv-fragments-images.sh
Comment (ampie) [reply src/main/bash/generate-csv-fragments-images.sh:6]: See above answer

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr66**

File: src/main/bash/copy-images.sh
Comment (ampie) [reply src/main/bash/copy-images.sh]: Yep. Fixing...

File: src/main/bash/copy-images.sh
Comment (w-caffiero-entando) [reply src/main/bash/copy-images.sh:9]: yep, you are right

**Review by w-caffiero-entando — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-k8s-controller-coordinator/pull/66#pullrequestreview-587359041" title="Approved">w-caffiero-entando</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-controller-coordinator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-k8s-controller-coordinator/pull/66#pullrequestreview-587359041" title="Approved">w-caffiero-entando</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-controller-coordinator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
